mixins/form.jsx

```diff
@@ -60,16 +66,15 @@ module.exports = {
     this.setState(newState);
     this.updateHeight();
   },
-  onAmountChange: function(name, value, amount) {
+  updateFormField: function(name, value, values) {
```

Comment:
This is now a generic function.

`updateFormField` is for props stored on the form mixin. `onchange` is a callback that does the similar thing, but used for handling props still stored in the components.